ds701_book/20-Recommender-Systems.qmd

```diff
@@ -362,39 +488,75 @@ These properties interfere with similarity assessment.
 Bias correction is crucial for CF recommender systems.
 
 We need to include
+
 * Per-user offset
 * Per-item offset
 * Global offset
 
+## Representing biases
+
 Hence we need to form a per-item bias of:
     
-$$ b_{ui} = \mu + \alpha_u + \beta_i $$
+$$ 
+b_{ui} = \mu + \alpha_u + \beta_i 
+$$
+
+where 
+
+* $\alpha_u$ is the per-user offset of user $u$ and 
+* $\beta_i$ is the per-item offset of item $i$.
 
-where $\alpha_u$ is the per-user offset of user $u$ and $\beta_i$ is the per-item offset of item $i$.
 
-How can we estimate the $\alpha$s, the $\beta$s, and the $\mu$?
+## Estimating biases
+
+::: {.content-visible when-profile="slides"}
+$$ 
+b_{ui} = \mu + \alpha_u + \beta_i 
+$$
+:::
+
+How can we estimate the $\alpha\textnormal{s}$, the $\beta\textnormal{s}$, and
```

Comment:
How can we estimate $\alpha_u$, $\beta_i$, and $\mu$ for each combination of user and item?